README.md

```diff
@@ -92,14 +92,37 @@ in the application's environment.
 You may wish to construct the application object on a signal
 when the Django project is "ready". You can use the `ready()`
 method of the `AppConfig` class in the `apps.py` module of a
-Django app.
-
+Django app. If you migrate before including the TrainingSchool object into your code,
+this way should work fine in development:
 ```python
 school = TrainingSchool(env={
     "PERSISTENCE_MODULE": "eventsourcing_django",
 })
 ```
 
+But usually you need migrations to run before creating the objects from database data
+and also put the created object into django app config:
+```python
+class TrainingSchoolConfig(AppConfig):
+    default_auto_field = "django.db.models.BigAutoField"
+    name = "<project_name>.training_school"
+
+    def ready(self):
+        call_command("migrate")
+        self.create_training_school()
+
+    def create_training_school(self):
+        training_school = TrainingSchool(
+            env={"PERSISTENCE_MODULE": "eventsourcing_django"}
```

Comment:
just one notes...
we ended up putting this in a settings file.
`# Event Sourcing
EVENT_SOURCING_SETTINGS = {
    "env": {"PERSISTENCE_MODULE": "eventsourcing_django", "IS_SNAPSHOTTING_ENABLED": "y", "DJANGO_DB_ALIAS": "default"}
}`

then in our

utility where we wanna instantiate the class, we dependency inject.
`class BillingUtility():
    def __init__(self):
        self.billing_event_application = BillingEventApplication(**settings.EVENT_SOURCING_SETTINGS)`

thank you for sharing these notes too!
wanted to provide our example as well.

Reply:
That's neat!